scripts/LEVEL15.cpp

```diff
@@ -842,28 +844,38 @@ struct tMyHandle {
   int handle;
 };
 
-tMyHandle MyObjects[] = {
+std::array<tMyHandle, 18> MyObjects = {
+    tMyHandle
```

Comment:
this initializer has 19 items for an 18-element array. what is `tMyHandle`?

Reply:
No, there are only 18 items because `tMyHandle` is the type for item 0. This hint allows the compiler to know that the others are also type `tMyHandle`.

Reply:
OH, i see, i wasn't visually parsing this correctly due to the newline. This is actually
```
std::array<tMyHandle, 18> MyObjects = { tMyHandle{"MatCen1Peg", -1}, {"MatCen2Peg", -1}, ... };
```
ok. that's great then, i'm fine with this.